AVsitter2/Plugins/AVcontrol/[AV]root-RLV.lsl

```diff
@@ -676,6 +680,32 @@ state running
             {
                 list data = llParseStringKeepNulls(msg, ["|"], []);
                 SITTERS = llParseStringKeepNulls(llList2String(data, 4), ["@"], []);
+                
+                integer index = llListFindList(CAPTIVES, [id]);
+                string new_pose = llList2String(data,1);
+                string original_pose = llList2String(CAPTIVE_POSES_FLAGS, index - 1);
```

Comment:
Shouldn't this line be inside the `if (index > -1)` part, at least to ensure the index is valid? (not important as LSL ignores most errors, but at least it will save an access if the index isn't valid)

Or even inside the `if (llList2Integer(...` block.